adapter-tcp/src/main/resources/tcpclient.md

```diff
@@ -41,11 +35,11 @@ Run a stdout activity named 'stdout-test', with definitions from activities/stdo
 
   [Additional parameters may need to be provided](../../../../driver-cql/src/main/resources/ssl.md).
 
-- **host** - this is the name to connect to (remote server IP address)
+- **host** - this is the name to bind to (local interface address)
   - default: localhost
   - dynamic: false
-- **port** - this is the name of the port to connect to (remote server port)
-  - default: 12345
+- **port** - this is the name of the port to listen on
+  - default: 8080
```

Comment:
We may want to verify this port with @jshook , it is a common one across many apps and tooling.

Reply:
reverting everything to 12345, if it's the default port for other drivers as well, it should be the same here.